code/game/machinery/bluespace_pond.dm

```diff
@@ -0,0 +1,68 @@
+// Calls update_icon() on the surrounding bluespace ponds
+// center: source turf to update around
+/proc/update_surrounding_bsp(var/turf/centre)
```

Comment:
Why is this a global proc? It's only used from 2 places in the object so putting it in the object would be better.